bitsail-connectors/connector-redis/src/test/resources/fake_to_redis_hash.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
not necessary to have this mostly duplicated file. We can change config after load the json

Reply:
this is different type Reader and Writer, one is Hash and the other is String, in the Hash test, we need 3 fields, and in the String test, we need 2 fields.